dart/test/flat_buffers_test.dart

```diff
@@ -155,6 +155,9 @@ class CustomAllocator extends Allocator {
 
 @reflectiveTest
 class BuilderTest {
+  late final throwsAssertionError = (() =>
+      throwsA(predicate((e) => e.toString().contains('Failed assertion'))))();
```

Comment:
change this to `throwsA(isA<AssertionError>())`

Reply:
done